Model-Docs/example-model.yaml

```diff
@@ -55,10 +91,58 @@ input_files:
           - name: date
             ontology: schema.org
             uri: https://schema.org/Date
+    #a file (or files) that can be used as input to the model. They all have the variables described above
+  - example: 
```

Comment:
Let me check my understanding: there will be an `input` (in this case `rainfall`) which may have any number of associated files. However each file would have the same format, but possibly different geospatial/temporal coverage. 

If my understanding is correct, this makes sense since actually in DCAT spatial/temporal coverage is set at the file level, not the dataset (or `input` in this case) level.

Reply:
Yes, this is correct. That way if you want to run your point model with different files for South Sudan, you may do so. I think this separates the input spec (only needed once) from the files that could be associated to it (many)